numpyro/distributions/continuous.py

```diff
@@ -1114,3 +1114,37 @@ def mean(self):
     def variance(self):
         var = (self.scale ** 2) * (np.pi ** 2) / 3
         return np.broadcast_to(var, self.batch_shape)
+
+
+@copy_docs_from(Distribution)
+class TruncatedPolyaGamma(Distribution):
+    truncation_point = 4.0
+    num_log_prob_terms = 11
+    num_gamma_variates = 20
+    assert num_log_prob_terms % 2 == 1
+
+    arg_constraints = {}
+    support = constraints.interval(0.0, truncation_point)
+
+    def __init__(self, batch_shape=(), validate_args=None):
```

Comment:
is it ok to have batch_shape as an arg like this?

Reply:
yup, it is okie.